packages/components/pages/f-loyalty/src/store/loyalty.module.js

```diff
@@ -73,12 +51,14 @@ export default {
             commit(ACTION_SET_GLOBAL_USER_ID, resolveGlobalUserID(authToken, logger, errorHandler));
         },
 
-        [ACTION_INITIALISE_LOYALTY]: ({ commit, dispatch }, {
-            brazeApiKey, authToken, logger, errorHandler
+        [ACTION_INITIALISE_LOYALTY]: ({ commit, dispatch, getters}, {
```

Comment:
This should fix lint checks:
```suggestion
        [ACTION_INITIALISE_LOYALTY]: ({ commit, dispatch, getters }, {
```